js/src/data/actions.js

```diff
@@ -523,6 +523,18 @@ export function* disconnectAllAccounts() {
 			type: TYPES.DISCONNECT_ACCOUNTS_ALL,
 		};
 	} catch ( error ) {
+		if (
+			error.errors[
+				`${ API_NAMESPACE }/rest-api/authorize`
+			]?.message?.includes(
+				'No token found associated with the client ID and user'
+			)
```

Comment:
Why not just` if ( error.errors[`${ API_NAMESPACE }/rest-api/authorize`] ) ` It seems more generic and catching wide range of present and future `rest-api/authorize` errors

Reply:
My intention was to only skip errors related to a missing token. The revoke endpoint in WPCOM can throw errors when the client is invalid or missing JP Headers. Should we skip those too?

Reply:
What I would avoid is to attach the error message ton the condition to trigger the error. So if we are going to catch all errors (I think we want to) I will just check if `error.errors[${ API_NAMESPACE }/rest-api/authorize]` I assume that if the key is present in the errors array. it means some error happened in the auth endpoint.

Reply:
Thanks for the discussion, let's go ahead and skip the errors for now, and if they cause any issues later, we can always make adjustments. I've made the updates here: [007190f](https://github.com/woocommerce/google-listings-and-ads/pull/2515/commits/007190f26e4851e5350b8aab64cc0a2bf0f40c96)